dynatrace/stackstate_checks/dynatrace/dynatrace.py

```diff
@@ -284,21 +384,22 @@ def _get_labels(self, dynatrace_component):
                 labels.append("managementZones:%s" % zone.get("name"))
         if dynatrace_component.entityId:
             labels.append(dynatrace_component.entityId)
-        if dynatrace_component.monitoringState:
+        if dynatrace_component.get('monitoringState'):
```

Comment:
Isn't this a schematics model anymore?

Reply:
yeah interestingly not. if I do access some properties which are not part of the model then it fails.